pkgs/dominikh/go-tools/staticcheck/registry.yaml

```diff
@@ -4,19 +4,21 @@ packages:
     repo_owner: dominikh
     repo_name: go-tools
     description: Go static analysis, detecting bugs, performance issues, and much more
-    asset: staticcheck_{{.OS}}_{{.Arch}}.tar.gz
+    asset: staticcheck_{{.OS}}_{{.Arch}}.{{.Format}}
+    format: tar.gz
```

Comment:
I don't think this update is needed.
If the archive format is different by OS, we have to parameterize the format, but in that case we don't have to parameterize the format.